src/app/components/AppNavigator/index.tsx

```diff
@@ -0,0 +1,41 @@
+import 'react-native-gesture-handler';
+import React from 'react';
+import { createStackNavigator } from '@react-navigation/stack';
+import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
+import Routes from '@constants/routes';
+import { tabNavigatorConfig, stackNavigatorConfig } from '@config/navigation';
+import { RootTabParamList, RootStackParamList } from '@interfaces/navigation';
+import BookList from '@app/screens/BookList';
+import BookDetail from '@app/screens/BookDetail';
+import Wishlist from '@components/Wishlist';
+import TabBarIcon from '@components/TabBarIcon';
+
+const stackNavigator = createStackNavigator<RootStackParamList>();
+const tabNavigator = createBottomTabNavigator<RootTabParamList>();
```

Comment:
Se we use them like the documentation says. They're not technically components, but still
```suggestion
const StackNavigator = createStackNavigator<RootStackParamList>();
const TabNavigator = createBottomTabNavigator<RootTabParamList>();
```